health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/pages/employee/PopInbox.js

```
@@ -2,75 +2,270 @@ import React, { Fragment, useState, useEffect } from "react";
+      <SearchJurisdiction boundaries={boundaries} jurisdiction={{
+        "boundaryType": hierarchyLevel,
+        "boundaryCodes": jurisdiction
+      }} onSubmit={onSearch} onClear={onClear} />
+
+      <div className="pop-inbox-wrapper-filter-table-wrapper">
+        <InboxFilterWrapper
+          options={activeFilter}
+          onApplyFilters={onFilter}
+          clearFilters={clearFilters}
+          defaultValue={selectedFilter !== '' && activeFilter ? { [Object.entries(activeFilter)?.[0]?.[0]]: Object.entries(activeFilter)?.[0]?.[1] } : null}
+        ></InboxFilterWrapper>
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)